src/clients/discord.ts

```diff
@@ -0,0 +1,131 @@
+import { AnyChannel, Client, Collection, Message, PartialMessage, TextChannel } from 'discord.js';
+import { linkedListNode, nodeInterface } from '../linkedList';
+import { conjoinedMsg } from '../messageObjects';
+import { genericPromiseError } from './bridge';
+import tmijs from 'tmi.js';
+import configFile from '../../config.json';
+import bridge from './bridge';
+
+const discordClient = new Client({ intents: ['GUILDS', 'GUILD_MESSAGES'] });
+const twitchClient: tmijs.Client = bridge.twitchClient as tmijs.Client;
+let targetDiscordChannel: TextChannel | undefined;
+
+const messageLinkdListInterface: nodeInterface = bridge.messageLinkdListInterface;
```

Comment:
One thing that can be done here is something like:
```js
import {messageLinkedListInterface, twitchMessageSearchCache, discordTwitchCacheMap, lastUserStateMsg} from './bridge'
```

That way it's all one statement

Reply:
At the moment, the Bridge file just returns a class instance, so we have to pull the variables from bridge.whatever; if we don't have it in a class like suggested above we cannot reassign, due to import statements always being a const.

Reply:
Yeah, never mind; we can just use an object. I was being stupid and just doing `export let z: number = 0;` ETC instead of putting it in an object (:. 
Will start moving Bridge from a class to an obj in a moment; (HYDRA TIME /j)

Reply:
After about 5 minutes of refactoring it now looks like this for the imports, it is honestly worse - 
```ts
import bridge, { configFile, genericPromiseError, manageMsgCache, twitchDelete } from './bridge';
const { messageLinkdListInterface, twitchMessageSearchCache, discordTwitchCacheMap, lastUserStateMsg } = bridge;
```

Reply:
Removed the destructuring, it is now one import. We have to keep the default import though in order to get the properties we need, unless if you want to move it over from default export to just a normal one

Reply:
Closing this now though, cause I personally believe the import has been cleaned up enough.